packages/strapi-generate-new/files/config/environments/staging/security.json

```diff
@@ -50,7 +50,8 @@
     "headers": [
       "Content-Type",
       "Authorization",
-      "X-Frame-Options"
+      "X-Frame-Options",
+      "Origin"
```

Comment:
`Origin` only in `staging`?

Reply:
Good catch! I forgot to save my changes in production as well. It's fixed!